src/tests/JIT/Regression/JitBlue/Runtime_106124/Runtime_106124.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.Arm._Sve
+{
+    public static partial class Program
+    {
+        [Fact]
+        public static void TestEntryPoint()
+        {
+            var vr19 = Vector128.CreateScalar(0L).AsVector();
```

Comment:
@SwapnilGaikwad could you please wrap this with a `Sve.IsSupported` check? Looks like this test is trying to run on unsupported hardware in CI.

Reply:
Done.